assets/data/companys.json

```diff
@@ -0,0 +1,26 @@
+[
```

Comment:
companies.json
spelling mistake ;)

Reply:
Also I guess you are missing few companies

Reply:
Companies present in UI MockUps are all added
Spelling issue is fixed